nearlib/test/integration.test.js

```diff
@@ -1,15 +1,15 @@
 const { Account, KeyPair, Near, InMemoryKeyStore } = require('../');
+const  { aliceAccountName, newAccountCodeHash, storageAccountIdKey, createFakeStorage, sleep } = require('./test-utils');
 const dev = require('../dev');
 const fs = require('fs');
-const aliceAccountName = 'alice.near';
-// every new account has this codehash
-const newAccountCodeHash = 'GKot5hBsd81kMupNCXHaqbhv3huEbxAFMLnpcX2hniwn';
-const storageAccountIdKey = 'dev_near_user';
 let nearjs;
 let account;
 let keyStore;
 
 beforeAll(async () => {
+    // To avoid nonce collisions with promise test on alice
+    await sleep(1000);
```

Comment:
If we are hitting this, users will too. Can we do something here? Can we make different accounts per test or something?

Reply:
My understanding is that this is needed to avoid clashes when creating different account for promises test.

However you raise a good point that we should have some way to handle this gracefully on nearlib side when user e.g. is using same app from multiple browser tabs.

Reply:
Ideally we should create more dev users. Right now `dev.js` is such a hack. We expose alice's key here and if someone withdraws all the money out of alice, we'll not be able to create accounts with tokens.
Instead we can create a faucet contract with tokens and it should spawn accounts instead of alice. While we don't expose the PK or we even put non-existing PK on it.

Reply:
Created #729